tests/formattedcode/data/common/manifests-expected.jsonlines

```diff
@@ -12,19 +12,19 @@
           "--package": true
         },
         "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
-        "output_format_version": "3.0.0",
+        "output_format_version": "3.1.0",
         "message": null,
         "errors": [],
         "warnings": [],
         "extra_data": {
           "system_environment": {
-            "operating_system": "linux",
+            "operating_system": "mac",
             "cpu_architecture": "64",
-            "platform": "Linux-5.15.0-89-generic-x86_64-with-glibc2.29",
-            "platform_version": "#99~20.04.1-Ubuntu SMP Thu Nov 2 15:16:47 UTC 2023",
-            "python_version": "3.8.10 (default, Nov 22 2023, 10:22:35) \n[GCC 9.4.0]"
+            "platform": "macOS-14.5-x86_64-i386-64bit",
+            "platform_version": "Darwin Kernel Version 23.5.0: Wed May  1 20:16:51 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T8103",
+            "python_version": "3.12.3 (main, Apr  9 2024, 08:09:14) [Clang 15.0.0 (clang-1500.3.9.4)]"
           },
-          "spdx_license_list_version": "3.22",
+          "spdx_license_list_version": "3.23",
```

Comment:
Revert changes in the header, if there are no PR related changes. Headers are ignored when comparing results, so this regeneration does not add anything.

Reply:
Just regenerated tests again, with making sure that the headers are not hindered by changes.

Reply:
This is still there in the changes. you need to revert these changes, or get rid of the headers entirely (do this in a seperate PR).